public/src/components/Icons.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import { Icon } from '../Helpers'
+
+export const Icons = {
+    GreyDash: <i className={Icon.DASH} />,
```

Comment:
It is better to name the "style" something like `NotAvailable`/`NoSubmission` instead of `GreyDash` to give it semantic meaning in the code. Then if we later decide to change it to another icon of some sort, we only need to change it here and not everywhere we use the not available or no submission.